docs/getting_started/flyte_fundamentals.md

```diff
@@ -52,7 +50,6 @@ cluster, see the {ref}`Deployment Guide <deployment>`.
 :hidden:
 
 tasks_and_workflows
-creating_flyte_project
 package_register
 run_schedule
 visualizing_artifacts
```

Comment:
I think we should stop calling it artifacts because we are about to introduce an artifacts service... might be a good time to change the name here...